source/install/cloud-changelog.md

```diff
@@ -3,13 +3,81 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-09-29](#release-2021-09-29)
 - [Release 2021-08-12](#release-2021-08-12)
 - [Release 2021-07-29](#release-2021-07-29)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
 - [Release 2021-06-16](#release-2021-06-16)
 - [Release 2021-06-02](#release-2021-06-02)
 
+## Release 2021-09-29
+
+### Highlights
+
+#### Permalink Previews
+ - Added support for permalink previews for posts in Mattermost.
```

Comment:
Focus on user benefit: Previews are generated to minimize context switching when sharing message links in Channels.